### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: css/modal.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.modal-container {
+  display: grid;
+  grid-template-rows: 12.4rem 8rem 2rem 5.5rem 3rem;
+}
+.hero-image {
+  width: 300px;
+  height: 350px;
+}
```
Comment (srishtigautam21) [inline css/modal.css:13]: use rem here

File: css/listing.css [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+.promo-img {
+  height: 450px;
+  width: 350px;
+  grid-column: 2/4;
+  grid-row: 3/4;
+}
+.promo-bg {
+  height: 450px;
+  width: 350px;
```
Comment (rutu14) [inline css/listing.css]: Use rems

**Review by rutu14 — APPROVED** [review]

File: css/modal.css (hunk 1 above)
Comment (Ranjeetk489) [reply css/modal.css:13]: thanks for pointing it out.. i have rectified this at other places too

File: css/listing.css (hunk 2 above)
Comment (Ranjeetk489) [reply css/listing.css]: thanks for reviewing, i have pushed the changes.

**Pull request merged**